qemu/tests/cfg/qemu_guest_agent.cfg

```diff
@@ -104,12 +104,16 @@
             gagent_guest_reboot_pattern = "Restarting system"
             Windows:
                 gagent_guest_reboot_pattern = ""
+            s390x:
+                gagent_guest_reboot_pattern = "System Reboot"
         - check_halt:
             gagent_check_type = halt
             # This config would be overriden in guest-os.cfg.
             gagent_guest_shutdown_pattern = "System halted"
             Windows:
                 gagent_guest_shutdown_pattern = ""
+            s390x:
+                gagent_guest_reboot_pattern = "System Halt"
```

Comment:
This is the shutdown pattern.

Reply:
@PaulYuuu, I edited this patch, please have a look

Reply:
@BohdanMar Please pay attention to 'gagent_guest_reboot_pattern' and 'gagent_guest_shutdown_pattern'.